### DISCUSSION TITLE
Docker container

### DISCUSSION THREAD

**Discussion by Petapton** [megastep/makeself#334] [Ideas]
Hello everyone, I wonder why there's no docker container for makeself. Is it a specific policy or just nobody implemented it? Would a PR be accepted?

**Comment by megastep** [comment]
Nothing against it per se, just wondering what the use case for it would be?

**Reply by Petapton** [reply]
I'm thinking of using it in CI/CD pipelines without having to install makeself on each agent. In that case a docker image will do the trick

**Reply by megastep** [reply]
Alternatively if your base image is a Linux distro like Ubuntu or Debian, you should be able to just install the package as part of your Dockerfile and be all set...

**Reply by Petapton** [reply]
Yes, that is viable if you already have an image doing other stuff. Otherwise you should create an image just for it. And that's the point.
In fact the question was rather about replacing the `makeself` command with `docker run makeself`.
